usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -6329,6 +6758,13 @@ CK_RV ep11tok_wrap_key(STDLL_TokData_t * tokdata, SESSION * session,
         goto done;
     }
 
+    rc = check_ab_wrap(tokdata, &sign_blob, &sign_blob_len, &sobj,
+                       key_obj, wrap_key_obj, mech);
+    if (rc != CKR_OK) {
+        TRACE_ERROR("AB wrapping check failed.\n");
```

Comment:
Why not adding the rc value to the trace output ...

Reply:
Done